src/arc/_listener.js

```diff
@@ -0,0 +1,23 @@
+let _ssm = require('./_ssm')
+
+module.exports = function _arcListener ({ inventory }, req, res) {
+  if (req.method.toLowerCase() === 'post') {
+    let body = ''
+
+    req.on('data', chunk => {
+      body += chunk.toString()
+    })
+
+    req.on('end', () => {
+      if (req.url === '/_arc/ssm') {
```

Comment:
I think we should check for the path in the conditional at the top of the listener, otherwise I think we could get into a situation that if someone accidentally `POST`s to a URL on this listener _other_ than `/_arc/ssm`, a response would never be issued?

Reply:
Fixed at https://github.com/architect/sandbox/commit/a8b4b140c9596925b4b495467fce66196ba37bee